.eslintrc.js

```diff
@@ -54,6 +54,12 @@ module.exports = {
       ignore: svgPresentationAttributes,
     }],
 
+    // Rules that conflict with Prettier
+    'space-before-function-paren': 'off',
+    'nonblock-statement-body-position': 'off',
+    'operator-linebreak': 'off',
+    'implicit-arrow-linebreak': 'off',
```

Comment:
This shouldn't be done by hand I think but with https://github.com/prettier/eslint-config-prettier